src/cms/utils/slug_utils.py

```diff
@@ -26,76 +61,77 @@ def generate_unique_slug(form_object, foreign_model):
     * :func:`cms.forms.events.event_translation_form.EventTranslationForm.clean_slug`
     * :func:`cms.forms.pois.poi_translation_form.POITranslationForm.clean_slug`
 
-    :param form_object: The form which contains the slug field
-    :type form_object: ~django.forms.Form
+    :param args: The supplied arguments list
+    :type args: list
 
-    :param foreign_model: If the form instance has a foreign key to another model (e.g. because it is a translation of
-                          a content-object), this paramaeter contains the model of the foreign related object.
-    :type foreign_model: ~django.db.models.Model
+    :param kwargs: The supplied keyword arguments
+    :type kwargs: dict
 
     :raises ~django.core.exceptions.ValidationError: When no slug is given and there is also no field which can be used
                                                      as fallback (either ``title`` or ``name``).
 
     :return: An unique slug identifier
     :rtype: str
     """
+    slug = kwargs["slug"]
+    foreign_model = kwargs["foreign_model"]
+    object_instance = kwargs["object_instance"]
+    fallback = kwargs["fallback"]
+    if args:
+        region = args[0]
+        language = args[1]
```

Comment:
Is there a specific reason to use `args` instead of `kwargs`? 
```suggestion
    region = kwargs.get("region")
    language = kwargs.get("language")
```

Reply:
No, I have no preferences in this case, we can take kwargs